UseSampling description typo in DataSourceSettings

ramonbgc opened the pull request:
DataSourceSettings on the UseSampling property description is not correctly "marked". It probably should be **DataSourceSettings** instead of **DataSourceSettings"
I'm not sure, checking pattern followed on document, if it should be **DataSourceSettings** or _DataSourceSettings_

azurecla commented:
Hi __@kelillus__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@kelillus : Thanks for your contribution to the Azure documentation! The author, @mgoedtel, has been notified to review your proposed change.

MGoedtel commented:
@kelillus thank you for your contribution to the article.  I have reviewed proposed change and accept.

MGoedtel commented:
#sign-off

ramonbgc committed:
UseSampling description typo in DataSourceSettings

DataSourceSettings on the UseSampling property description is not correctly "marked". It probably should be **DataSourceSettings** instead of **DataSourceSettings"
I'm not sure, checking pattern followed on document, if it should be **DataSourceSettings** or _DataSourceSettings_
